scripts/dev_setup.py

```diff
@@ -25,13 +25,15 @@ def exec_command(command):
 print('Running dev setup...')
 print('Root directory \'{}\'\n'.format(root_dir))
 
+# install general requirements
+exec_command('pip install -r requirements.txt')
```

Comment:
Ok then, so I can move the requirement.txt restore back to the bottom but specifically install setuptools==30.4.0 at the beginning, does it sound good?